Source/IntegrationTests/TestFiles/LitTests/LitTest/verification/proofDivision/isolatePaths.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: ! %verify --progress VerificationJobs --cores=1 %s &> %t.raw
+// RUN: ! %verify --progress Batch --cores=1 %s &> %t.raw
```

Comment:
Same, here, if it's a breaking change, please document it in docs/dev/news at least.